src/views/TableDetail.vue

```diff
@@ -164,23 +183,45 @@ export default Vue.extend({
       return result;
     },
   },
+
+  asyncComputed: {
+    columnTypes: {
+      async get() {
+        try {
+          return await api.tableColumnTypes(this.workspace, this.table);
+        } catch (err) {
+          return {};
+        }
+      },
+
+      default: {},
+    },
+  },
+
   watch: {
-    workspace() {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    workspace(this: any) {
```

Comment:
Why is this now required in way more places?

Reply:
Haha, I'm definitely susceptible to nerd sniping, too. Your resolution seems appropriate for now.